frontend/playground-frontend/src/pages/dashboard.tsx

```diff
@@ -103,17 +112,18 @@ const BlankGrid = () => {
   const router = useRouter();
   return (
     <div id="blank-grid-wrapper">
-      <div id="blank-grid">
-        <p>
-          You haven't trained any models yet. Create your first model below!
-        </p>
-        <button
-          id="blank-grid-button"
-          onClick={() => router.push({ pathname: "/train"})}
-        >
-          Train Model
-        </button>
-      </div>
+      <Upsell
```

Comment:
@anikethswain for the PR description, please attach an image/screenshot of the upsell?

Reply:
Here's a screenshot:
![image](https://github.com/DSGT-DLP/Deep-Learning-Playground/assets/41639552/4e5d4283-6e31-4098-b38d-60395adbc78c)